src/rust/bitbox02/src/memory.rs

```diff
@@ -16,16 +16,18 @@
 // deduct one for the null terminator.
 pub const DEVICE_NAME_MAX_LEN: usize = bitbox02_sys::MEMORY_DEVICE_NAME_MAX_LEN as usize - 1;
 
+pub struct Error {}
```

Comment:
Can just be a unit struct? There's nothing to store in it.
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=9e3e97d945e0d3ada821802c00b12e13

Reply:
oh cool didn't know. done